docs/developer/developer.adoc

```diff
@@ -0,0 +1,357 @@
+= OpenSCAP Developer Manual
+:oscap_git: https://github.com/OpenSCAP/openscap
+:toc: preamble
+:numbered:
+
+This part of documentation is meant to serve mainly to developers who want to
+contribute to OpenSCAP, help to fix bugs, or take an advantage of
+the OpenSCAP library and create own projects on top of it.
+
+toc::[]
+
+== Building OpenSCAP on Linux
+If you want to build the ```libopenscap``` library and the ```oscap``` tool from
+the {oscap_git}[source code] then follow these instructions:
+
+. *Get the source code*
++
+Choose *1a* or *1b* depending on whether you want sources from a release tarball or the git repository.
+
+.. Use a release tarball:
+
+ # replace ${version} with the desired version
+ wget https://github.com/OpenSCAP/openscap/releases/download/${version}/openscap-${version}.tar.gz
+ tar -xzpf openscap-${version}.tar.gz
+ cd openscap-${version}
++
+**OR**
+
+.. Use fresh sources from git repository.
+
+ $ git clone https://github.com/OpenSCAP/openscap.git
+ $ cd openscap
++
+
+. *Get the build dependencies*
++
+--
+To build the library you will also need the following build dependencies.
+Build dependencies may vary depending on enabled features (by the ```cmake``` command).
+By default, OpenSCAP requires these dependencies to build:
```

Comment:
I think that the list of dependencies is not needed, it spans huge amount of vertical space, individual items have platform-dependent names, and we repeat everything below, where we tell people how to install those.

Reply:
+1